webapp/src/shared/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,53 @@
+import './Avatar.css';
+
+import { Icon, IconVariant, IconSize } from '../index';
+import { Color } from '../../../shared/types';
```

Comment:
should this be `import { Color } from '../../shared/types';` ??

Reply:
not, because `shared` folder is three levels under this path. But I see what you mean and we could have `../../types`

that being said I'll keep it as is because seeing this is imported from a shared folder gives more information